ui/rogue/subtlety/presets.ts

```diff
@@ -6,34 +6,36 @@ import { SavedTalents } from '../../core/proto/ui';
 import { Stats } from '../../core/proto_utils/stats';
 import SubtletyApl from './apls/subtlety.apl.json';
 import P1SubtletyGear from './gear_sets/p1_subtlety.gear.json';
+import P2SubtletyGear from './gear_sets/p2_subtlety.gear.json';
```

Comment:
We should probably follow Blizzard Phases, so this should be Phase 3. Other specs have already called it P3 as well.